manifests/00-rbac-role.yaml

```diff
@@ -1,40 +1,59 @@
 kind: Role
-apiVersion: rbac.authorization.k8s.io/v1beta1
+apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: console-operator
   namespace: openshift-console
 rules:
 - apiGroups:
   - console.openshift.io
   resources:
-  - "*"
+  - consoles
   verbs:
-  - "*"
+  - get
+  - list
+  - watch
+  - create
+  - update
+  - delete
```

Comment:
Add TODO to drop all the deletes

Reply:
added (above, line 7, as it applies to many)